pydantic/_internal/_decorators.py

```diff
@@ -309,7 +309,9 @@ def inspect_validator(validator: Callable[..., Any], mode: FieldValidatorModes)
     )
 
 
-def inspect_field_serializer(serializer: Callable[..., Any], mode: Literal['plain', 'wrap']) -> tuple[bool, bool]:
+def does_field_serializer_require_info_arg(
```

Comment:
This function returns two things.

Reply:
Good point. It’ll be nice when we can do `-> {“foo”: int}`.